Showcase/Application/Core/Components/Badge/BadgeModel.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Vitamin iOS
+//  Apache License 2.0
+//
+
+import VitaminCore
+
+enum BadgeModel {
+    struct Section: Identifiable {
+        var id: String {
+            name
+        }
+        let name: String
+        let items: [Item]
+    }
+
+    struct Item: Identifiable {
+        var id: String {
+            "\(value ?? 0)-\(variant.name)"
+        }
+        var value: Int?
+        var variant: VitaminBadgeVariant = .standard
+    }
+
+    static let sections: [BadgeModel.Section] = {
+        var varientSection: [BadgeModel.Section] = []
```

Comment:
Just a little typo....
```suggestion
        var variantSection: [BadgeModel.Section] = []
```

Reply:
🙈